openedx/features/partners/tests/test_g2a_registration.py

```diff
@@ -0,0 +1,107 @@
+from ddt import ddt, data
+
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+
+from openedx.features.partners.tests.factories import PartnerFactory, OrganizationFactory
+from openedx.core.lib.api.test_utils import ApiTestCase
+from lms.djangoapps.onboarding.models import Organization
+
+
+@ddt
+class RegistrationViewTest(ApiTestCase):
+    """Tests for the registration"""
```

Comment:
Let's add proper documentation, it is only stating the obvious

Reply:
done